browser/go/conversion-test/conversion_test.go

```diff
@@ -0,0 +1,451 @@
+		registerValidationResultsField(description, "accepted | fillableTakerAssetAmount")
+		registerValidationResultsField(description, "accepted | isNew")
+	}
+
+	registerValidationResultsField(description, "rejected | length")
+	for i := 0; i < rejectedLength; i++ {
+		registerValidationResultsField(description, "rejected | orderHash")
+		registerValidationResultsField(description, "rejected | signedOrder | chainId")
+		registerValidationResultsField(description, "rejected | signedOrder | makerAddress")
+		registerValidationResultsField(description, "rejected | signedOrder | takerAddress")
+		registerValidationResultsField(description, "rejected | signedOrder | senderAddress")
+		registerValidationResultsField(description, "rejected | signedOrder | feeRecipientAddress")
+		registerValidationResultsField(description, "rejected | signedOrder | exchangeAddress")
+		registerValidationResultsField(description, "rejected | signedOrder | makerAssetData")
+		registerValidationResultsField(description, "rejected | signedOrder | makerAssetAmount")
+		registerValidationResultsField(description, "rejected | signedOrder | makerFeeAssetData")
+		registerValidationResultsField(description, "rejected | signedOrder | makerFee")
+		registerValidationResultsField(description, "rejected | signedOrder | takerAssetData")
+		registerValidationResultsField(description, "rejected | signedOrder | takerAssetAmount")
+		registerValidationResultsField(description, "rejected | signedOrder | takerFeeAssetData")
+		registerValidationResultsField(description, "rejected | signedOrder | takerFee")
+		registerValidationResultsField(description, "rejected | signedOrder | expirationTimeSeconds")
+		registerValidationResultsField(description, "rejected | signedOrder | salt")
+		registerValidationResultsField(description, "rejected | signedOrder | signature")
+		registerValidationResultsField(description, "rejected | kind")
+		registerValidationResultsField(description, "rejected | status | code")
+		registerValidationResultsField(description, "rejected | status | message")
+	}
+}
+
+func registerGetOrdersResponseField(description string, field string) {
+	registerTest(fmt.Sprintf("(getOrdersResponseTest | %s | %s)", description, field))
+}
+
+func registerOrderEventField(description string, field string) {
+	registerTest(fmt.Sprintf("(orderEventTest | %s | %s)", description, field))
+}
+
+func registerSignedOrderField(description string, field string) {
+	registerTest(fmt.Sprintf("(signedOrderTest | %s | %s)", description, field))
+}
+
+func registerStatsField(description string, field string) {
+	registerTest(fmt.Sprintf("(statsTest | %s | %s)", description, field))
+}
+
+func registerValidationResultsField(description string, field string) {
+	registerTest(fmt.Sprintf("(validationResultsTest | %s | %s)", description, field))
+}
+
+func registerTest(test string) {
+	testCases = append(testCases, fmt.Sprintf("\"%s: true\"", test))
```

Comment:
nit: I think using backticks is a little cleaner.

```suggestion
	testCases = append(testCases, fmt.Sprintf(`"%s: true"`, test))
```